robot-endpoint/camera.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import cv2 as cv
+
+
+cap = cv.VideoCapture(0)
+if not cap.isOpened():
+    print("Cannot open camera")
+    exit()
+
+frame_width = int(cap.get(3))
+frame_height = int(cap.get(4))
+while True:
+    # Capture frame-by-frame
+    ret, frame = cap.read()
+    # if frame is read correctly ret is True
+    if not ret:
+        print("Can't receive frame (stream end?). Exiting ...")
+        break
+    # Our operations on the frame come here
+    gray = cv.cvtColor(frame, cv.COLOR_BGR2GRAY)
+    # Display the resulting frame
+    cv.imshow('frame', gray)
+    if cv.waitKey(1) == ord('q'):
+        break
+# When everything done, release the capture
+cap.release()
+cv.destroyAllWindows()
+
+
+# import cv2
```

Comment:
is this commented out for future use? or was this an old implementation

Reply:
This was working in an old version but suddenly stopped working